test/extended/networking/kubevirt/client.go

```diff
@@ -93,6 +95,65 @@ func (c *Client) GetJSONPath(resource, name, jsonPath string) (string, error) {
 	}
 	return strings.TrimSuffix(strings.TrimPrefix(output, `"`), `"`), nil
 }
+
+func (c *Client) GetPodsByLabel(labelKey, labelValue string) ([]string, error) {
+	output, err := c.oc.AsAdmin().Run("get").Args("pods", "-n", c.oc.Namespace(), "-l", fmt.Sprintf("%s=%s", labelKey, labelValue), "-o", "jsonpath={.items[*].metadata.name}").Output()
```

Comment:
You can use `-o name` instead of the jsonpath also do we have to care of running vs completed pods during migration we can end up returning more pods than needed?

Reply:
>You can use -o name instead of the jsonpath

ACK

>also do we have to care of running vs completed pods during migration we can end up returning more pods than needed?

@maiqueb also raised this [here](https://github.com/openshift/origin/pull/30197#discussion_r2321228532) - I think that right now we should keep it simple (we only check restarted VMs, not migrated ones), but having said that - since both of you suggested it - if you still think it's needed to be added now - I'll do it. your call.